catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -589,8 +575,24 @@ public void dispose() {
 		disposeStageButKeepActors();
 		font.dispose();
 		axes.dispose();
+
 		disposeTextures();
 		disposeClonedSprites();
+
+		SoundManager.getInstance().clear();
+		PhysicsShapeBuilder.getInstance().reset();
+		embroideryList = null;
+		if (penActor != null) {
+			penActor.dispose();
+		}
+	}
+
+	public void finish() {
+		if (CameraManager.getInstance() != null) {
+			CameraManager.getInstance().setToDefaultCamera();
+		}
```

Comment:
I'm not sure if there could be problems if I move the CameraManager code to the dispose method. Maybe with the FaceDetection feature?
To be honest, I haven't really looked into that. I guess it doesn't matter where the Camera is reset…